packages/backend/src/peripherals/database/migrations/014_forced_transactions.ts

```diff
@@ -0,0 +1,36 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+import { up as createForcedTransactionEvents } from './012_forced_transaction_events'
```

Comment:
Is there a reason up is used, but down is reimplemented with `dropTableIfExists`?

Reply:
Good catch - removed.

Reply:
Why not just import down too?

Reply:
Another good catch :D